### ISSUE TITLE
How to use the linux release?

### ISSUE THREAD

**Issue by luxpir** [GSGBen/t2md#1]
Tried to run under wine, and in a VM, where it couldn't access ```coreclr.dll``` - any guidance on getting started? Will be fine to follow the readme after that.

Ah, running the Windows build under WINE now and the README info is wrong - should be ```output-folder``` not ```output-path```. Then it creates the ```t2md.json``` config file in the dir you ran the exe in, not the output folder.

I also grabbed my token from the link provided by Trello vs the URL you suggested, that worked.

Ran the command again and it seems to have dumped everything, it just ended with an error encountered so I'm not 100% confident it's all there. Will have a look manually, but also try another folder to compare in the VM if I can get it working there.

Basically... just needs the README updating a little, but otherwise great work, and thanks!

PS - An option to have all content from a card in a single file would be interesting too.

**Comment by GSGBen** [comment]
Hi,

`t2md` within the linux zip is a native linux binary, so once you've extracted it you can mark it executable and run it directly, you shouldn't need to use the windows version:

```
chmod +x t2md
./t2md
```

I've updated the readme instructions to correct the parameter name too, thanks.

**Comment by luxpir** [comment]
Ah great, appreciated. Gah, my bad for not making it executable first. I do it with .sh files all the time so not sure why I missed that here. Thanks anyway. It output exactly the same as the VM version did (number of files, size on disk) but both ended with an error:

```
Unhandled exception. System.Text.Json.JsonException: The JSON value could not be converted to System.Double. Path: $.actions[347].data.card.pos | LineNumber: 0 | BytePositionInLine: 313936.
 ---> System.InvalidOperationException: Cannot get the value of a token type 'String' as a number.
   at System.Text.Json.Utf8JsonReader.TryGetDouble(Double& value)
   at System.Text.Json.Utf8JsonReader.GetDouble()
   at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.Converters.IEnumerableDefaultConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.JsonPropertyInfo`1.ReadJsonAndSetMember(Object obj, ReadStack& state, Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   --- End of inner exception stack trace ---
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& state, Utf8JsonReader& reader, Exception ex)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[TValue](JsonConverter jsonConverter, Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[TValue](Utf8JsonReader& reader, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, JsonSerializerOptions options)
   at GoldenSyrupGames.T2MD.Cli.ProcessTrelloBoardAsync(TrelloApiBoardModel trelloApiBoard, CliOptions options)
   at GoldenSyrupGames.T2MD.Cli.RunAsync(CliOptions options)
   at GoldenSyrupGames.T2MD.Cli.Main(String[] args)
   at GoldenSyrupGames.T2MD.Cli.<Main>(String[] args)
```

Just chucked the whole thing in there in case any of it useful to you. Same error on both Win and Lin though. 

Something in one of my boards not exporting neatly? Do you think it's copied everything cleanly anyway?

**Comment by GSGBen** [comment]
Hmmm, that's strange. It looks like one of your card positions (usually e.g. 123.45) is in the .json export as a string, or otherwise isn't cleanly being converted across to a double. I'll have to look into it. Are you able to provide sanitised copies of the .json files that were downloaded into the output folder? That would help. If so and you don't want to post them publicly, please email then to ben <at> goldensyrupgames.com.

**Comment by luxpir** [comment]
Quite a few long json files I won't have time to sanitise. Thanks for offering to look though. 

Anything I can grep them for to help?

**Comment by GSGBen** [comment]
All good, #3 is the same issue and has some examples. I'll fix it there.
